### ISSUE TITLE
FOG Installation Problems on Windows 10

### ISSUE THREAD

**Issue by JasmineTF** [FOGProject/fogproject#316]
I have windows 10 version 1809, build 17763.557
I have 32-bit Ubuntu 16.04 and Fog version 1.5.6
I have Windows 10 in legacy boot mode.

When started, the pxe initiates normally and, when it sees there are no FOG commands, the white menu box appears with the default highlighted command being to boot from the hard drive, just like it normally does. The problem is that a black screen appears with a blinking cursor and remains indefinitely. There is no boot to the hard drive. If I boot and manually cancel the pxe boot with a keystroke, it has no problem booting from the hard drive. The problem seems to be with FOG correctly accessing the hard drive from its boot menu. Any ideas what to do?

**Comment by Sebastian-Roth** [comment]
@JasmineTF Sorry for the late reply. Have been away for a bit an still try to catch up with it all. Just for you to know - there are more active people in the [forums](https://forums.fogproject.org) and usually you get a way quicker response there.

Have you tried playing with the so called "exit types" in FOG yet? Go to a host's settings in the web UI and see the BIOS EXIT TYPE (and as well UEFI EXIT TYPE). Set different options here and see which one works for your hardware.

**Comment by JasmineTF** [comment]
Thank you for responding, Sebastian!
I discovered that for option #67, I had undionly.kpxe, which works for windows 7.  However, for Windows 10, I had to change it to ipxe.efi.
That change did the trick!  :)

    On Monday, July 1, 2019, 3:13:24 PM CDT, Sebastian-Roth <notifications@github.com> wrote:  
 
 
@JasmineTF Sorry for the late reply. Have been away for a bit an still try to catch up with it all. Just for you to know - there are more active people in the forums and usually you get a way quicker response there.

Have you tried playing with the so called "exit types" in FOG yet? Go to a host's settings in the web UI and see the BIOS EXIT TYPE (and as well UEFI EXIT TYPE). Set different options here and see which one works for your hardware.
